tests/msbuild/Xamarin.MacDev.Tasks.Tests/TaskTests/LocalizationStringTest.cs

```diff
@@ -96,66 +96,6 @@ private string TranslateError (string culture, string errorCode)
 		IList<string> commonIgnoreList = null;
 		static string shortCommonPath = "xamarin-macios/tests/msbuild/Xamarin.MacDev.Tasks.Tests/TaskTests/LocalizationIgnore/common-Translations.ignore";
 
-		[SetUp]
-		public void SetUp ()
-		{
-			commonIgnoreList = ReadFile ($"{Directory.GetCurrentDirectory ()}/TaskTests/LocalizationIgnore/common-Translations.ignore");
-		}
-
-		[TestCase ("cs-CZ")]
-		[TestCase ("de-DE")]
-		[TestCase ("es-ES")]
-		[TestCase ("fr-FR")]
-		[TestCase ("it-IT")]
-		[TestCase ("ja-JP")]
-		[TestCase ("ko-KR")]
-		[TestCase ("pl-PL")]
-		[TestCase ("pt-BR")]
-		[TestCase ("ru-RU")]
-		[TestCase ("tr-TR")]
-		[TestCase ("zh-CN")]
-		[TestCase ("zh-TW")]
-		public void AllErrorTranslation (string culture)
-		{
-			StringBuilder errorList = new StringBuilder (string.Empty);
-			IList<string> cultureIgnoreList = null;
-			List<string> commonValidEntries = new List<string> ();
-			List<string> cultureValidEntries = new List<string> ();
-
-			string fullCulturePath = $"{Directory.GetCurrentDirectory ()}/TaskTests/LocalizationIgnore/{culture}-Translations.ignore";
-			string shortCulturePath = $"xamarin-macios/tests/msbuild/Xamarin.MacDev.Tasks.Tests/TaskTests/LocalizationIgnore/{culture}-Translations.ignore";
-			CultureInfo originalUICulture = Thread.CurrentThread.CurrentUICulture;
-			CultureInfo originalCulture = Thread.CurrentThread.CurrentCulture;
-
-			cultureIgnoreList = ReadFile (fullCulturePath);
-
-			foreach (var errorCodeInfo in typeof (MSBStrings).GetProperties ()) {
-				try {
-					var errorCode = errorCodeInfo.Name;
-					if (errorCode == "ResourceManager" || errorCode == "Culture")
-						continue;
-					string englishError = TranslateError ("en-US", errorCode);
-					string newCultureError = TranslateError (culture, errorCode);
-
-					if (commonIgnoreList.Contains (errorCode)) {
-						Assert.AreEqual (englishError, newCultureError, $"{errorCode} is translated. Remove {errorCode} from {shortCommonPath}");
-						commonValidEntries.Add (errorCode);
-					} else if (cultureIgnoreList.Contains (errorCode)) {
-						Assert.AreEqual (englishError, newCultureError, $"{errorCode} is translated. Remove {errorCode} from {shortCulturePath}");
-						cultureValidEntries.Add (errorCode);
-					} else if (englishError == newCultureError)
-						errorList.Append ($"{errorCode} ");
-				} finally {
-					Thread.CurrentThread.CurrentUICulture = originalUICulture;
-					Thread.CurrentThread.CurrentCulture = originalCulture;
-				}
-			}
-
-			Assert.IsEmpty (errorList.ToString (), $"The following errors were not translated. Add them to {shortCommonPath} or {shortCulturePath}");
-			Assert.IsEmpty (cultureIgnoreList.Except (cultureValidEntries), $"Not all error codes in {shortCulturePath} are valid or are repeated. Please remove.");
-			Assert.IsEmpty (commonIgnoreList.Except (commonValidEntries), $"Not all error codes in {shortCommonPath} are valid or are repeated. Please remove.");
-		}
-
 		IList<string> ReadFile (string path)
```

Comment:
This does not look like it is longer used.

Reply:
Good call!